test/game_teams_collection_test.rb

```diff
@@ -31,6 +31,22 @@ def test_find_id_of_teams_with_worst_fans
     assert_equal [24, 5, 19], @game_teams_collection.worst_fans_team_id
   end
 
+  def test_most_accurate_team
+    first_game = @game_teams_collection.all_games_by_season("20122013").first
+    last_game = @game_teams_collection.all_games_by_season("20122013").last
+    season_2017 = @game_teams_collection.all_games_by_season("20172018")
+
+    assert_equal "2012", first_game.game_id.to_s[0..3]
+    assert_equal "2012", last_game.game_id.to_s[0..3]
+    assert_equal 2, @game_teams_collection.most_accurate_team_id("20122013")
+    assert_equal true, season_2017.all? {|game| game.game_id.to_s[0..3]}
+  end
+
```

Comment:
Nice job testing and making sure everything is working the way its all supposed to.